src/Authentication/Authentication/AdalTokenProvider.cs

```diff
@@ -107,23 +121,11 @@ public AdalTokenProvider(Func<IServicePrincipalKeyStore> getKeyStore)
             switch (credentialType)
             {
                 case AzureAccount.AccountType.User:
-                    return userTokenProvider.GetAccessToken(
-                        config,
-                        promptBehavior,
-                        promptAction,
-                        userId,
-                        password,
-                        credentialType);
+                    return _userTokenProvider.GetAccessToken(config, promptBehavior, promptAction, userId, password, credentialType);
```

Comment:
I don't remember changing this. It might have happened automatically when I renamed the field with the underscore. This is why I like the idea of having a code style that is applied to the entire codebase automatically. 😉

Reply:
As a ganeral rule, this is good, but this can run afoul of other engineering best practices (e.g. it is a good idea to make changes to legacy codebases minimal, to prevent introducing regressions)